stories/IssueArchives.vue

```diff
@@ -0,0 +1,210 @@
+<template>
+    <div class="issue-archives">
+        <div class="issue-archives__tabs">
+            <ul>
+                <li v-for="tab in archives" :key="tab.year">
+                    <button
+                        @click="activetab = tab.year"
+                        :aria-selected="activetab === tab.year"
+                        :id="'issue-archive-button-' + tab.year"
+                    >
+                        {{ tab.year }}
+												<span class="issue-archives__button--volume">
+													—
+													Volume {{ tab.volume }}
+												</span>
+                    </button>
+                </li>
+            </ul>
+        </div>
+        <div class="issue-archives__tabcontent">
+            <section class="tab">
+                <template v-for="tab in archives">
+                    <section
+                        v-if="activetab === tab.year"
+                        :key="tab.year"
+                        :id="'tab' + tab.year"
+                        :aria-labelledby="'issue-archive-button-' + tab.year"
+                    >
+                    <issue-summary
+                        v-for="issueSummary in tab.issues"
+                        :key="issueSummary.volume"
+                        v-bind="issueSummary"
+                    />
+                    </section>
+                </template>
+            </section>
+        </div>
+    </div>
+</template>
+
+<script>
+import IssueSummary from './IssueSummary.vue';
+
+export default {
+    components: { IssueSummary },
+    props: {
+        archives: Object
+    },
+    data() {
+        return {
+            activetab: 0
+        };
+    },
+    mounted() {
+        this.activetab = this.archives[0].year;
```

Comment:
As a general practice, it's usually better to track a whole object and then use what you need when you need it. So in this case, I would change this line to:

```js
this.activeTab = this.archives[0];
```

Then in the template above, you can use the following:

```html
<button
	@click="activeTab = tab"
	:aria-selected="activeTab.year === tab.year"
	:id="'issue-archive-button-' + tab.year"
>
	...
</button>
```

If, down the road, you decide you want to know more about the active tab, you'll have access to this through `this.activeTab`.